src/shogun/evaluation/Calibration.h

```diff
@@ -64,26 +70,44 @@ namespace shogun
 		* @param targets The true labels corresponding to the predictions
 		* @return boolean indicating whether the calibration was succesful
 		**/
-		virtual bool fit_binary(CBinaryLabels* predictions, CBinaryLabels* targets);
+		virtual bool fit_binary(CBinaryLabels* predictions, CBinaryLabels* targets)
```

Comment:
i still dont get why do we need default implementation with error

Reply:
I'm not sure about making all methods pure virtual... suppose a calibration method does not support multiclass calibration, then the implementation will have to derive `fit_multiclass` and give error message manually. Would that be fine?